test/win32/test.rc

```diff
@@ -76,14 +76,14 @@ BEGIN
         BEGIN
             VALUE "Comments", "LokiNET test suite"
             VALUE "CompanyName", "Loki Foundation"
-            VALUE "FileDescription", "LokiNET for Microsoft� Windows� NT�"
+            VALUE "FileDescription", "LokiNET for Microsoft� Windows� NT�"
 #ifdef LLARP_RELEASE_MOTTO
             VALUE "FileVersion", VERSION_STRING(0.4.0, RELEASE_MOTTO, GIT_REV)
 #else
-            VALUE "FileVersion", VERSION_STRING(0.4.0-dev-, GIT_REV)
```

Comment:
what objection is there to mark the build revision in binaries? this is useful for debug tracking

Reply:
For whatever reason, this was flagged as breakage here: https://travis-ci.org/loki-project/loki-network/builds/560207652#L1614

Reply:
gah, escaping strikes again